version.json

```diff
@@ -1,7 +1,6 @@
 {
   "version": "15.1-preview",
   "publicReleaseRefSpec": [
-    "^refs/heads/master$",
-    "^refs/heads/xplat$"
+    "^refs/heads/.*-rtw"
```

Comment:
Not sure this'll last forever (I assume at some point we'll have something like `vs15-update1` that's a release branch) but good enough for now.